cypress/helpers/util.ts

```diff
@@ -42,62 +54,17 @@ export const mermaidUrl = (
   return url;
 };
 
-export const imgSnapshotTest = (
-  graphStr: string,
-  _options: CypressMermaidConfig = {},
-  api = false,
-  validation?: any
-): void => {
-  cy.log(JSON.stringify(_options));
-  const options: CypressMermaidConfig = Object.assign(_options);
-  if (!options.fontFamily) {
-    options.fontFamily = 'courier';
-  }
-  if (!options.sequence) {
-    options.sequence = {};
-  }
-  if (!options.sequence || (options.sequence && !options.sequence.actorFontFamily)) {
-    options.sequence.actorFontFamily = 'courier';
-  }
-  if (options.sequence && !options.sequence.noteFontFamily) {
-    options.sequence.noteFontFamily = 'courier';
-  }
-  options.sequence.actorFontFamily = 'courier';
-  options.sequence.noteFontFamily = 'courier';
-  options.sequence.messageFontFamily = 'courier';
-  if (options.sequence && !options.sequence.actorFontFamily) {
-    options.sequence.actorFontFamily = 'courier';
-  }
-  if (!options.fontSize) {
-    options.fontSize = 16;
-  }
-
-  const url: string = mermaidUrl(graphStr, options, api);
-  openURLAndVerifyRendering(url, options, validation);
-};
-
-export const urlSnapshotTest = (
-  url: string,
-  _options: CypressMermaidConfig,
-  _api = false,
-  validation?: any
-): void => {
-  const options: CypressMermaidConfig = Object.assign(_options);
-  openURLAndVerifyRendering(url, options, validation);
-};
-
-export const renderGraph = (
-  graphStr: string,
-  options: CypressMermaidConfig = {},
-  api = false
-): void => {
-  const url: string = mermaidUrl(graphStr, options, api);
-  openURLAndVerifyRendering(url, options);
-};
-
-export const openURLAndVerifyRendering = (
+/**
+ * opens the {@link url}, then verifies that the `window` is rendered,
+ * and the `svg` is visible.
+ *
+ * @param url -
+ * @param options -
+ * @param validation -
```

Comment:
You can remove all the @params without any valid descriptions.

Reply:
I added them as a placeholder so we could discuss what should we add.

Reply:
I added them as a placeholder so we could discuss what should we add.